README.md

```diff
@@ -76,6 +76,34 @@ example below will be announced as `Homepage`.
 <h1 aria-label="Homepage">Project Title</h1>
 ```
 
+## Focus
+
+On page navigation, the plugin will reset focus to the body, imitating default browser behavior.
+When clicking anchor links on the same page, the plugin will focus the target of the link.
+
+> [!TIP]
+> When linking to other sections on the page, make sure to target a single item that is descriptive
+> of the section, e.g. a heading or a button. Linking to a large container has multiple issues:
+> Some browsers will scroll to the center of the targeted container, disorienting near-sighted users
+> with increased zoom. Screen readers will also start reading out the content of the targeted
+> element, which can be overwhelming when a whole content section is being read out.
+
+## Styling
+
+Browsers will display a visible outline around elements focussed by this plugin. That's great for
```

Comment:
Stumbled upon `focussed` with two s. Seems like `focused` is more common, but with two ss it's still correct. As you prefer :)

Reply:
No opinion either way, changed it to focused with a single s.